app/src/main/java/org/fossasia/openevent/Services/BookmarkAlarmService.java

```diff
@@ -0,0 +1,68 @@
+package org.fossasia.openevent.Services;
+
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.app.Service;
+import android.content.Intent;
+import android.media.RingtoneManager;
+import android.os.IBinder;
+import android.support.v4.app.NotificationCompat;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.activities.SessionDetailActivity;
+import org.fossasia.openevent.data.Session;
+import org.fossasia.openevent.dbutils.DbSingleton;
+import org.fossasia.openevent.utils.IntentStrings;
+
+/**
+ * Created by Manan Wason on 21/08/15.
+ */
+public class BookmarkAlarmService extends Service {
+
+    private NotificationManager mManager;
+
+    @Override
+    public IBinder onBind(Intent arg0) {
+        return null;
+    }
+
+    @Override
+    public void onCreate() {
+        super.onCreate();
+    }
+
+    @Override
+    public int onStartCommand(Intent intent, int flags, int startId) {
+        handleStart(intent, startId);
+        return START_NOT_STICKY;
+
+
+    }
+
+    @Override
+    public void onDestroy() {
+        super.onDestroy();
+    }
+
+    void handleStart(Intent intent, int startId) {
+        mManager = (NotificationManager) this.getApplicationContext().getSystemService(this.getApplicationContext().NOTIFICATION_SERVICE);
+        int id = intent.getIntExtra(IntentStrings.SESSION, 0);
+        DbSingleton dbSingleton = DbSingleton.getInstance();
+        Session session = dbSingleton.getSessionById(id);
+        Intent intent1 = new Intent(this.getApplicationContext(), SessionDetailActivity.class);
+        intent1.putExtra(IntentStrings.SESSION, session.getTitle());
+        PendingIntent pendingNotificationIntent = PendingIntent.getActivity(this.getApplicationContext(), 0, intent1, PendingIntent.FLAG_UPDATE_CURRENT);
+
+        NotificationCompat.Builder mBuilder = new NotificationCompat.Builder(this)
+                .setSmallIcon(R.mipmap.ic_launcher)
+                .setContentTitle("session about to start")
```

Comment:
Fix the string. Use a localized version. Also put more info like the time etc. and not just the title of the session.